integration_tests/test_ibc.py

```diff
@@ -0,0 +1,250 @@
+        os.popen(cmd)
+        dstaddr = f"{coin_receiver}"
+        olddstbalance = get_balance(cronos, dstaddr, dst_denom)
+        time.sleep(5)
+        newdstbalance = get_balance(cronos, dstaddr, dst_denom)
+        expectedbalance = olddstbalance + src_amount * (10 ** 10)
+        assert expectedbalance == newdstbalance
+        break
+    assert True
+    pass
+
+
+def test_ibc_reverse(cronos, chainmain, hermes):
+    # wait for hermes
+    time.sleep(20)
+    while True:
+        my_ibc0 = "chainmain-1"
+        my_ibc1 = "cronos_777-1"
+        my_channel = "channel-0"
+        my_config = hermes.configpath
+        # signer21
+        coin_receiver = "cro1u08u5dvtnpmlpdq333uj9tcj75yceggszxpnsy"
+        src_amount = 2 * (10 ** 10)
+        src_denom = "basetcro"
+        dst_denom = "ibc/6B5A664BF0AF4F71B2F0BAA33141E2F1321242FBD\
+5D19762F541EC971ACB0865"
+        # dstchainid srcchainid srcportid srchannelid
+        # chainmain-1 <- cronos_777-1
+        cmd = f"hermes -c {my_config} tx raw ft-transfer \
+        {my_ibc0} {my_ibc1} transfer {my_channel} {src_amount} \
+        -o 1000 -n 1 -d {src_denom} -r {coin_receiver} -k testkey"
+        os.popen(cmd)
+        dstaddr = f"{coin_receiver}"
+        olddstbalance = get_balance(chainmain, dstaddr, dst_denom)
+        time.sleep(5)
+        newdstbalance = get_balance(chainmain, dstaddr, dst_denom)
+        expectedbalance = olddstbalance + src_amount
+        assert expectedbalance == newdstbalance
+        break
+    assert True
+    pass
+
+
+def test_contract(cronos, chainmain, hermes):
+    cronos_node = "http://127.0.0.1:26701"
+    cronos_chainid = 777
+    cronos_gas = 3000000000
+    cronos_mnemonics = "night renew tonight dinner shaft scheme \
+domain oppose echo summer broccoli agent face guitar surface \
+belt veteran siren poem alcohol menu custom crunch index"
+    web3api = Web3(Web3.HTTPProvider(cronos_node))
+    web3api.eth.account.enable_unaudited_hdwallet_features()
+    account = web3api.eth.account.from_mnemonic(cronos_mnemonics)
+    contract_creator_address = account.address
+    web3api.eth.get_balance(contract_creator_address)
+    """
+    #install_solc(version='latest')
+    #compiled_sol = compile_source(
+        pragma solidity >0.5.0;
+
+        contract Greeter {
+            string public greeting;
+
+            constructor() public {
+                greeting = 'Hello';
+            }
+
+            function setGreeting(string memory _greeting) public {
+                greeting = _greeting;
+            }
+
+            function greet() view public returns (string memory) {
+                return greeting;
+            }
+        }
+    # prepare contract
+    #contract_id, contract_interface = compiled_sol.popitem()
+    #bytecode = contract_interface['bin']
+    #abi = contract_interface['abi']
+    """
+
+    # precompiled contract
+    bytecode = (
+        "608060405234801561001057600080fd5b50604051806040016040528060"
```

Comment:
How do you update the compiled bytecode, I suggest to put the contract to `integration_tests/contracts`, and `npx hardhat typechain` will compile all, you can consult other integration test cases: https://github.com/crypto-org-chain/cronos/blob/main/integration_tests/test_gravity.py#L104

Reply:
ok